solution-manifest.yaml

```diff
@@ -1,6 +1,6 @@
 id: SO0050
 name: media2cloud-on-aws
-version: 3.1.5
+version: 3.1.6
```

Comment:
@justindra  This will cause an issue later when upstream makes another update and also uses "3.1.6". I should be able to deploy this to our environments by just overwriting the current 3.1.5 on S3, so it would likely be best to not include this version change in our fork. Thoughts? CloudFormation will still accept the updated template from the same path.

Reply:
@benb1n I had issues with Cloudformation not updating the lambda function as it thinks it's still the same version. Should we do a `3.1.5-patch-1` instead for the version? That way it should still allow for any upstream changes to be useable.

Reply:
@justindra Oh, ok. Yeah, I think a patch suffix would be better.